COCOAPODS.md

```diff
@@ -96,39 +94,62 @@ the build process of an Xcode project.
+ * [A Pod library with custom C Interop options](#add-a-dependency-on-a-pod-library-with-custom-c-interoperability-options)
+ * [A static Pod library](#add-a-dependency-on-a-static-pod-library)
+
+A Kotlin project requires the `pod()` function call in `build.gradle.kts` (`build.gradle`) for adding a Pod dependency. Each dependency requires a separate function call.
+The function has a configuration block where you could specify the parameters for the dependency.
+
+When you add a new dependency and re-import the project in IntelliJ IDEA the new dependency will be added automatically.
+There are no additional steps required. 
 
-[Complete the initial configuration](#install-the-cocoapods-dependency-manager-and-plugin), and when you add a new 
-dependency and re-import the project in IntelliJ IDEA; the new dependency will be added automatically. There are no 
-additional steps required.
+To use your Kotlin project with Xcode, you should [make changes in your project Podfile](#update-podfile-for-xcode).
 
 ### Add a dependency on a Pod library from the CocoaPods repository
 
-1. Add dependencies on a Pod library that you want to use from the CocoaPods repository with `pod()`  to `build.gradle.kts` 
-(`build.gradle`) of your project.  
-    > You can also add dependencies on subspecs.
-    {:.note}                                                                                                                                                              >
+You can add dependencies on a Pod library from the CocoaPods repository with `pod()` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+ 
+1. Specify the name of a Pod library in the `pod()` function. In the configuration block you can specify the version of the library using `version` parameter. To use the latest version of the library, omit the `version` parameter.
 
-    <div class="sample" markdown="1" theme="idea" data-highlight-only>
-    
-    ```kotlin
-    kotlin {
-        ios()
-    
-        cocoapods {
-            summary = "CocoaPods test library"
-            homepage = "https://github.com/JetBrains/kotlin"
-            pod("AFNetworking", "~> 4.0.0")
-            
-            pod("SDWebImage/MapKit")
-        }
-    }
-    ```
-    
-    </div>
+   > You can add dependencies on subspecs.
+   {:.note}
+
+2. Specify the minimum deployment target version for the Pod library.
+
+   > If you don't specify the minimum deployment target version and a dependency Pod requires a higher deployment target, you may get an error.
+   {:.note}
+
+   <div class="sample" markdown="1" theme="idea" data-highlight-only>
+   
+   ```kotlin
+   kotlin {
+       ios()
+
+       cocoapods {
 
-2. Re-import the project.
+           ios.deploymentTarget = "13.5"
+
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           pod("AFNetworking") {
+                version = "~> 4.0.1"
+           }
```

Comment:
No `SDWebImage` anymore, so it should probably be removed from the imports below.

Reply:
I removed them.